resources/views/settings/account.blade.php

```diff
@@ -131,9 +131,15 @@
                     <p class="mt-3">
                         {{ __('profiles.sections.delete.warning') }}
                     </p>
+                    <p class="mt-3">
+                        {{ __('profiles.sections.delete.goodbye') }}
+                    </p>
 
                     <div class="py-5">
                         {!! Form::model($user, ['method' => 'PATCH', 'route' => ['settings.account.destroy']]) !!}
+                        <div class="form-group required">
+                        {!! Form::text('goodbye',null, ['class' => 'form-control']) !!}
```

Comment:
You can add in the `[]` array `['class' => 'form-control', 'required']`. This is an html property which will stop the form from being submitted as long as a required field isn't filled out :)